## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 423994`

Logs: https://github.com/06kellyjac/nixpkgs-review-gha/actions/runs/16192929321


---
### `x86_64-linux` (sandbox = true)
<details>
  <summary>:fast_forward: 1 package blacklisted:</summary>
  <ul>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>conftest</li>
    <li>nixpkgs-manual</li>
  </ul>
</details>


---
### `aarch64-linux` (sandbox = true)
<details>
  <summary>:fast_forward: 1 package blacklisted:</summary>
  <ul>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>conftest</li>
    <li>nixpkgs-manual</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>conftest</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>nixpkgs-manual</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>conftest</summary>
<pre>Building subPackage ./parser/properties
Building subPackage ./parser/spdx
Building subPackage ./parser/textproto
Building subPackage ./parser/toml
Building subPackage ./parser/vcl
Building subPackage ./parser/xml
Building subPackage ./parser/yaml
Building subPackage ./plugin
Building subPackage ./policy
Building subPackage ./runner
buildPhase completed in 1 minutes 42 seconds
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
ok  	github.com/open-policy-agent/conftest/builtins	0.025s
ok  	github.com/open-policy-agent/conftest/document	0.021s
--- FAIL: TestDownloadFailsWhenFileExists (0.00s)
    downloader_test.go:26: listen tcp 127.0.0.1:0: bind: operation not permitted
FAIL
FAIL	github.com/open-policy-agent/conftest/downloader	0.044s
FAIL</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>conftest</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>nixpkgs-manual</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>conftest</summary>
<pre>Building subPackage ./parser/properties
Building subPackage ./parser/spdx
Building subPackage ./parser/textproto
Building subPackage ./parser/toml
Building subPackage ./parser/vcl
Building subPackage ./parser/xml
Building subPackage ./parser/yaml
Building subPackage ./plugin
Building subPackage ./policy
Building subPackage ./runner
buildPhase completed in 58 seconds
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
ok  	github.com/open-policy-agent/conftest/builtins	0.035s
ok  	github.com/open-policy-agent/conftest/document	0.017s
--- FAIL: TestDownloadFailsWhenFileExists (0.00s)
    downloader_test.go:26: listen tcp 127.0.0.1:0: bind: operation not permitted
FAIL
FAIL	github.com/open-policy-agent/conftest/downloader	0.032s
FAIL</pre>
</details>
</details>

--- a/pkgs/by-name/co/conftest/package.nix
+++ b/pkgs/by-name/co/conftest/package.nix
@@ -13,6 +13,8 @@ buildGoModule (finalAttrs: {
   pname = "conftest";
   version = "0.62.0";
 
+  __darwinAllowLocalNetworking = true; # required for tests
+
   src = fetchFromGitHub {
     owner = "open-policy-agent";
     repo = "conftest";

